install.sh

```diff
@@ -23,13 +24,15 @@ esac
 
 deno_uri="https://github.com/denoland/deno/releases/download/${version}/deno_${os}_x64.gz"
 
-curl -fL# -o "$deno_dir/deno.gz" "$deno_uri"
-gunzip -df "$deno_dir/deno.gz"
-chmod +x "$deno_dir/deno"
+curl -fL# -o "$exe.gz" "$deno_uri"
+gunzip -df "$exe.gz"
+chmod +x "$exe"
 
-echo "Deno was installed successfully."
+echo "Deno was installed successfully to $exe"
```

Comment:
The `.` at the end of the sentence was removed. The next sentence ends with a `.`, too. So for consistency, I would re-add the `.`.

Reply:
yes, makes it more copy & paste-able